--- simplefile.php	2025-05-08 21:06:26.033892278 -0300
+++ simplefile_bad1.php	2025-05-08 21:08:10.864427594 -0300
@@ -69,7 +69,7 @@
 stream_set_blocking($stdin, false);
 
 
-function restoreNameFromUserMap(array &$entry, array $userMap): void {
+function restoreNameFromUserMap(array $entry, array $userMap): void {
     // Restore receiver_name from userMap
     if (isset($userMap[(string)($entry["receiver_id"] ?? "")])) {
         $entry["receiver_name"] =
@@ -294,39 +294,6 @@
                             LOCK_EX
                         );
 
-                        /*
-                        // Restore receiver_name from userMap
-                        if (isset($entry["receiver_id"])) {
-                            $entry["receiver_name"] =
-                                $userMap
-                                [(string)($entry["receiver_id"] ?? "")]
-                                ?? "";
-
-                            if ($entry["receiver_name"] === "") {
-                                unset($entry["receiver_name"]);
-                            }
-                        }
-
-                        // Restore sender_name from userMap
-                        $entry["sender_name"] =
-                            $userMap[(string)($entry["sender_id"] ?? "")]
-                            ?? "guest";
-
-                        $json = json_encode($entry);
-
-                        if ($json !== false) {
-                            echo $json . PHP_EOL;
-
-                            flush();
-                            fflush(STDOUT);
-                        } else {
-                            fwrite(
-                                STDERR,
-                                "JSON encode error (output): "
-                                . json_last_error_msg() . PHP_EOL
-                            );
-                        }
-                        */
                         // Restore {receiver,sender}_name from userMap
                         restoreNameFromUserMap($entry, $userMap);
                     }
@@ -362,41 +329,8 @@
 
                     if (!is_array($entry)) continue 1;
 
-                    /**/
-                    // Restore receiver_name from userMap
-                    if (isset($entry["receiver_id"])) {
-                        $entry["receiver_name"] =
-                            $userMap[(string)($entry["receiver_id"] ?? "")]
-                            ?? "";
-
-                        if ($entry["receiver_name"] === "") {
-                            unset($entry["receiver_name"]);
-                        }
-                    }
-
-                    // Restore sender_name from userMap
-                    $entry["sender_name"] =
-                        $userMap[(string)($entry["sender_id"] ?? "")]
-                        ?? "guest";
-
-                    $json = json_encode($entry);
-
-                    if ($json !== false) {
-                        echo $json . PHP_EOL;
-
-                        flush();
-                        fflush(STDOUT);
-                    } else {
-                        fwrite(
-                            STDERR,
-                            "JSON encode error (output): "
-                            . json_last_error_msg() . PHP_EOL
-                        );
-                    }
-
-                    /**/
                     // Restore {receiver,sender}_name from userMap
-                    //restoreNameFromUserMap($entry, $userMap);
+                    restoreNameFromUserMap($entry, $userMap);
                 }
 
                 fclose($fh);
